src/server/pfs/server/server.go

```diff
@@ -88,7 +80,13 @@ func NewBlockAPIServer(dir string, cacheBytes int64, backend string, etcdAddress
 			return nil, err
 		}
 		return blockAPIServer, nil
+	case LocalBackendEnvVar:
+		blockAPIServer, err := newLocalBlockAPIServer(dir, cacheBytes, etcdAddress)
+		if err != nil {
+			return nil, err
+		}
+		return blockAPIServer, nil
 	default:
-		return NewLocalBlockAPIServer(dir)
+		return nil, fmt.Errorf("unrecognized storage backend")
```

Comment:
Why not leave this as the default? You should always be able to construct a local backend and, all else being equal I'd rather not add an extra way for deployments to fail.